src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -70,21 +65,15 @@ public function indexAction(Request $request): Response
         $helperListConfiguration = $this->buildListConfiguration(
             'id_feature',
             'id_feature',
+            //@todo: position update is still handled by legacy ajax controller action. Need to handle in dedicated PR
```

Comment:
topic for another PR. see AdminController::processPosition() and AdminFeatureController::ajaxProcessUpdatePositions()